midp/gfx.js

```diff
@@ -187,15 +189,16 @@ var currentlyFocusedTextEditor;
                 img.src = '';
                 resolve();
             }
-            img.onerror = function(e) {
-                URL.revokeObjectURL(img.src);
-                img.src = '';
-                reject(new JavaException("java/lang/IllegalArgumentException", "error decoding image"));
-            }
-        });
-    }, true);
+            //img.onerror = function(e) {
```

Comment:
Probably forgot to uncomment this one. We should add a test for a image decoding error